components/script/dom/element.rs

```diff
@@ -20,7 +20,7 @@ use dom::bindings::codegen::Bindings::HTMLTemplateElementBinding::HTMLTemplateEl
 use dom::bindings::codegen::Bindings::NamedNodeMapBinding::NamedNodeMapMethods;
 use dom::bindings::codegen::Bindings::NodeBinding::NodeMethods;
 use dom::bindings::codegen::InheritTypes::{CharacterDataCast, DocumentDerived, ElementCast};
-use dom::bindings::codegen::InheritTypes::{ElementDerived, EventTargetCast, HTMLAnchorElementCast};
+use dom::bindings::codegen::InheritTypes::{ElementDerived, EventTargetCast, HTMLAnchorElementCast, HTMLFieldSetElementDerived, HTMLLegendElementDerived, HTMLOptGroupElementDerived};
```

Comment:
https://travis-ci.org/servo/servo/jobs/84430060

The tidy check failed because these imports aren't sorted